Further increase the F3 GMessage buffer size

The buffer size for handling F3 GMessages off pubsub was bumped a couple of times before activation. We see reports that some folks still see `subscriber too slow` errors, e.g. reported [here](https://filecoinproject.slack.com/archives/CP50PPW2X/p1746609149125029).


The channel buffer size change is not a consensus breaking change. The root cause for the need for larger buffer is slow calls to Lotus to get power table, which is being addressed as part of https://github.com/filecoin-project/go-f3/issues/961.

However, in the meantime or additionally, it makes sense to bump the buffer size further just to be on the safe side. The increase in memory footprint is small since GMessages are small. Therefore this is a low risk investment to make sure no GMessages are getting dropped by pubsub before getting processed.

@ribasushi also kindly pointed out these errors on his node:

```
2025-05-07T09:20:35.397Z        WARN    pubsub  go-libp2p-pubsub@v0.13.0/tag_tracer.go:157      error bumping delivery tag: unable to bump decaying tag for peer 12D3KooWBmT4dsUNj1oAad4RnSUx47mhGTWpjYu9u4x1TnS5ydZY, tag pubsub-deliveries:/f3/granite/0.0.3/filecoin, delta 1; queue full (len=128)
2025-05-07T09:20:35.397Z        WARN    pubsub  go-libp2p-pubsub@v0.13.0/tag_tracer.go:157      error bumping delivery tag: unable to bump decaying tag for peer 12D3KooWLNXrYFz9k3jQxwN7FrP8rbqoKgACSbtAscnK21JofPie, tag pubsub-deliveries:/f3/granite/0.0.3/filecoin, delta 1; queue full (len=128)
2025-05-07T09:20:35.433Z        WARN    pubsub  go-libp2p-pubsub@v0.13.0/tag_tracer.go:157      error bumping delivery tag: unable to bump decaying tag for peer 12D3KooWKS2KqiGjew96FFNaXprhv16kwkNLvuJjcu73GbCSFbE5, tag pubsub-deliveries:/f3/granite/0.0.3/filecoin, delta 1; queue full (len=128)
2025-05-07T09:20:35.433Z        WARN    pubsub  go-libp2p-pubsub@v0.13.0/tag_tracer.go:157      error bumping delivery tag: unable to bump decaying tag for peer 12D3KooWGznqfes5NP4nFW3hZLK1nDSmDy5K9L7YVFPWuTqRwjQr, tag pubsub-deliveries:/f3/granite/0.0.3/filecoin, delta 1; queue full (len=128)
2025-05-07T09:20:35.434Z        WARN    pubsub  go-libp2p-pubsub@v0.13.0/tag_tracer.go:157      error bumping delivery tag: unable to bump decaying tag for peer 12D3KooWNVSsbhLoTUyTxfQ9Ft2U7Bqi6TuCJ42Gm5T6TeiGJUy4, tag pubsub-deliveries:/f3/granite/0.0.3/filecoin, delta 1; queue full (len=128)
2025-05-07T09:20:35.434Z        WARN    pubsub  go-libp2p-pubsub@v0.13.0/tag_tracer.go:157      error bumping delivery tag: unable to bump decaying tag for peer 12D3KooWMkbBWvZ5dbubEPqwi8yGLSP7wgTBD6LjGura2TgKAYqj, tag pubsub-deliveries:/f3/granite/0.0.3/filecoin, delta 1; queue full (len=128)
2025-05-07T09:20:35.435Z        WARN    pubsub  go-libp2p-pubsub@v0.13.0/tag_tracer.go:157      error bumping delivery tag: unable to bump decaying tag for peer 12D3KooWNroy3LUJfPU5tVpz2p7Mca2PAkUjjESmjfySJTTxKM1H, tag pubsub-deliveries:/f3/granite/0.0.3/filecoin, delta 1; queue full (len=128)
2025-05-07T09:20:35.435Z        WARN    pubsub  go-libp2p-pubsub@v0.13.0/tag_tracer.go:157      error bumping delivery tag: unable to bump decaying tag for peer 12D3KooWR8S1h1xqZYqnac42A3FMjBzdP5cLGdPoHxUK92bBWTGh, tag pubsub-deliveries:/f3/granite/0.0.3/filecoin, delta 1; queue full (len=128)
2025-05-07T09:20:35.436Z        WARN    pubsub  go-libp2p-pubsub@v0.13.0/tag_tracer.go:157      error bumping delivery tag: unable to bump decaying tag for peer QmYfbkDM2cJMH5TpqGUD4LuCUyAFCWjqbhHmdtFyQxKNav, tag pubsub-deliveries:/f3/granite/0.0.3/filecoin, delta 1; queue full (len=128)
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -21,7 +21,8 @@
 - fix(f3): limit the concurrency of F3 power table calculation ([filecoin-project/lotus#13085](https://github.com/filecoin-project/lotus/pull/13085))
 - feat(f3): remove dynnamic manifest functionality and use static manifest ([filecoin-project/lotus#13074](https://github.com/filecoin-project/lotus/pull/13074))
 - chore(deps): bump filecoin-ffi for fvm@v4.7 which adds Logs and IpldOps to debug FVM execution traces ([filecoin-project/lotus#13029](https://github.com/filecoin-project/lotus/pull/13029))
-- chore: return `method not supported` via Gateway when /v2 isn't supported by the backend ([filecoin-project/lotus#13121](https://github.com/filecoin-project/lotus/pull/13121)
+- chore: return `method not supported` via Gateway when /v2 isn't supported by the backend ([filecoin-project/lotus#13121](https://github.com/filecoin-project/lotus/pull/13121))
+- chore: increase the F3 GMessage buffer size to 1024 ([filecoin-project/lotus#13126](https://github.com/filecoin-project/lotus/pull/13126))
 
 See https://github.com/filecoin-project/lotus/blob/release/v1.33.0/CHANGELOG.md
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -44,7 +44,7 @@ For the set of changes since the last stable release:
 
 # Node and Miner v1.32.2 / 2025-04-04
 
-This Lotus v1.32.2 release is a **MANDATORY patch release**. After the Calibration network upgraded to nv25, a bug was discovered in the ref-fvm KAMT library affecting ERC-20 token minting operations. You can read the full techincal breakdown of the issue [here](https://github.com/filecoin-project/builtin-actors/pull/1667).
+This Lotus v1.32.2 release is a **MANDATORY patch release**. After the Calibration network upgraded to nv25, a bug was discovered in the ref-fvm KAMT library affecting ERC-20 token minting operations. You can read the full technical breakdown of the issue [here](https://github.com/filecoin-project/builtin-actors/pull/1667).
 
 This patch release includes the following updates:
 - Schedules a mandatory Calibration upgrade, happening on `2025-04-07T23:00:00Z`, to fix the ERC-20 token minting bug on the Calibration network.

--- a/build/buildconstants/f3manifest_mainnet.json
+++ b/build/buildconstants/f3manifest_mainnet.json
@@ -39,7 +39,7 @@
   "PubSub": {
     "CompressionEnabled": true,
     "ChainCompressionEnabled": true,
-    "GMessageSubscriptionBufferSize": 768,
+    "GMessageSubscriptionBufferSize": 1024,
     "ValidatedMessageBufferSize": 1024
   },
   "ChainExchange": {

